.jazzy.yaml

```diff
@@ -1,13 +1,11 @@
 clean: true
 author: RevenueCat
 author_url: https://revenuecat.com
-objc: true
 sdk: iphonesimulator
-module: Purchases
-umbrella_header: Purchases/Public/Purchases.h
 module_version: 3.13.0-SNAPSHOT
 github_url: https://github.com/revenuecat/purchases-ios
 github_file_prefix: https://github.com/revenuecat/purchases-ios/tree/3.13.0-SNAPSHOT
 output: docs
-# Leaving this commented out. We used to specify this before, but now it's working without it
-# xcodebuild_arguments: [--objc,Purchases/Public/Purchases.h,--,-x,objective-c,-isysroot,$(xcrun --show-sdk-path),-I,$(pwd)]
```

Comment:
these docs haven't been relevant for a while, and they weren't particularly informative